src/draw/lv_draw_private.h

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file lv_draw_private.h
+ *
+ */
+
+#ifndef LV_DRAW_PRIVATE_H
+#define LV_DRAW_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+struct lv_draw_task_t {
```

Comment:
make sure all of these fields have functions that are able to set them. otherwise it will be unusable to bindings.

Reply:
We have added the required API.